eventRouting/eventRouting_test.go

```diff
@@ -67,4 +76,107 @@ var _ = Describe("Events", func() {
 		})
 	})
 
+	Context("Test Filter", func() {
+		var testMsg *events.Envelope
+		var logMessage *events.LogMessage
+		var eventType events.Envelope_EventType
+		BeforeEach(func() {
+			eventType = events.Envelope_LogMessage
+			logMessage = &events.LogMessage{}
+			testMsg = &events.Envelope{
+				EventType:  &eventType,
+				LogMessage: logMessage,
+			}
+			stats.ConsumeLogMessage = uint64(0)
+			stats.Publish = uint64(0)
+			stats.Ignored = uint64(0)
+		})
+		Context("When the log message is consumed and published", func() {
+			BeforeEach(func() {
+				eventRouting.RouteEvent(testMsg)
+			})
+			It("Should have one more Log Message consumed", func() {
+				Ω(stats.ConsumeLogMessage).Should(Equal(uint64(1)))
+			})
+			It("Should have one more event published", func() {
+				Ω(stats.Publish).Should(Equal(uint64(1)))
+			})
+			It("Should have 0 event ignored", func() {
+				Ω(stats.Ignored).Should(Equal(uint64(0)))
+			})
+		})
+		Context("When the log message has cf_ignored_app field", func() {
+			BeforeEach(func() {
+				var appId = "Fake App"
+				logMessage = &events.LogMessage{AppId: &appId}
+				testMsg = &events.Envelope{
+					EventType:  &eventType,
+					LogMessage: logMessage,
+				}
+				caching.GetAppStub = func(string) (*App, error) {
+					return &App{IgnoredApp: true}, nil
+				}
+				eventRouting.RouteEvent(testMsg)
+			})
+			It("Should have one more Log Message consumed", func() {
+				Ω(stats.ConsumeLogMessage).Should(Equal(uint64(1)))
+			})
+			It("Should have one more event published", func() {
+				Ω(stats.Publish).Should(Equal(uint64(0)))
+			})
+			It("Should have 0 event ignored", func() {
+				Ω(stats.Ignored).Should(Equal(uint64(1)))
+			})
+		})
+	})
+
+	Context("When we specify which orgs log message we publish", func() {
```

Comment:
Should be about excluding orgs since we don't know in advance all the orgs that will be submitting logs and that list is constantly growing.

Reply:
It's ok, we want to include not exclude.